python-avd/pyavd/_eos_designs/schema/schema_fragments/default_connected_endpoints_description.yml

```
@@ -14,10 +14,10 @@ keys:
       Default description or description template to be used on all ports to connected endpoints.
       This can be a template using the AVD string formatter syntax: https://avd.arista.com/stable/ansible_collections/arista/avd/roles/eos_designs/docs/how-to/custom-descriptions-names.html#avd-string-formatter-syntax.
       The available template fields are:
-        - `endpoint_type`: The `type` from `connected_endpoints_keys` like `server`, `router` etc.
+        - `endpoint_type` - the `type` of the connected endpoint either set on the endpoint or taken from `connected_endpoints_keys.type` like `server`, `router` etc.
```

Should this item use the same style?

```suggestion
        - `endpoint_type`: The `type` of the connected endpoint either set on the endpoint or taken from `connected_endpoints_keys.type` like `server`, `router` etc.
```